### PR TITLE
Upgrade kubebuilder to 1.0.3.

### PR DESCRIPTION
**Pull request by gyliu513**
/sig multicluster

/cc @font @madhusudancs @perotinus @pmorie

### CODE REVIEW

**Comment by font** [comment]
@gyliu513 can you separate generated code from non-generated code into distinct commits?

**Comment by gyliu513** [comment]
@font all of the code are generated automatically via kubebuilder 1.0.3 by following steps at https://github.com/kubernetes/cluster-registry/blob/master/docs/development.md#updating-generated-codedocs , and I just update this PR by separating the fix for `gofmt` which was not automatically generated by `kubebuilder`, hope it is OK.

File: docs/development.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ You must install [kubebuilder](https://github.com/kubernetes-sigs/kubebuilder)
 in order to work in this repository. `kubebuilder` is in active development, and
 we expect that the structure of this repository will need to change as
 `kubebuilder` is improved. All of the commands below are verified to work with
-`kubebuilder` `0.1.12`.
+`kubebuilder` [1.0.3](https://github.com/kubernetes-sigs/kubebuilder/releases/tag/v1.0.3).
```
Comment (font) [inline docs/development.md]: (no action required) @gyliu513 I was referring to this doc update in the same commit.

**Comment by font** [comment]
@gyliu513 Can you check the failed job?

**Comment by gyliu513** [comment]
Ping @font for a /lgtm label

**Pull request merged**